tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Models/ResultTests.cs

```diff
@@ -54,5 +95,44 @@ public void TwoDifferenceResults_AreNotEqual()
 
             actual.Should().BeFalse();
         }
+
+        [Test]
+        public void TwoDifferentResultsT_AreNotEqual()
+        {
+            var success = Result.Success("TestA");
+            var failure = Result.Success("TestB");
+
+            var actual = success.Equals(failure);
+
+            actual.Should().BeFalse();
+        }
+
+        [Test]
+        public void ToResult_ConvertSuccessResultT_ReturnsSuccessResult()
+        {
+            var sut = Result.Success("Test");
```

Comment:
I'm not a fan of `sut` – possibly another for the next code standards meeting!

Personally, I'd prefer something more descriptive, perhaps `genericResult` but leave as is if you want given we currently have no standards covering this (as far as I'm aware).

Reply:
Yeah it is on my list to add for the coding standards meeting. I will apply the standard after the meeting tomorrow 👍